apps/dashboard/tests/global.setup.ts

```diff
@@ -0,0 +1,33 @@
+import path from 'path';
+import { clerk, clerkSetup } from '@clerk/testing/playwright';
+import { test as setup } from '@playwright/test';
+
+const authFile = path.join(__dirname, '../playwright/.clerk/user.json');
+
+setup('global setup', async ({ page }) => {
+  await clerkSetup();
+
+  /**
+   * TODO: Currently we are running tests with a single Clerk user and organization.
+   * This approach has a drawback that the tests are not independent from each other and write the data to the same organization.
+   * We should consider creating a new Clerk user and organization for each test using the @clerk/backend package.
+   * Then initialize the BE session with the new Clerk user and organization and update Clerk user metadata from the newly created session.
+   */
```

Comment:
hopefully, it explains the problem well

Reply:
Correct! We should leverage [Clerk Backend API](https://clerk.com/docs/reference/backend-api), create a user before every run, sign in with that user, and then delete the user at the end.

Regarding the need for Clerk webhooks during user and organization creation, I suggest connecting to Mongo directly from the test code using the response of the Clerk API to create the user and organization record the application needs to function.

On user deletion, I don't think we need to do anything in Mongo as we run in a dockerized environment.

Reply:
I'll think about this, as it will require reimplementing all the current setup.